packages/components/src/vue/components/form-components/input-autocomplete.vue

```diff
@@ -92,6 +93,11 @@ export default {
       this.getAutocompleteWords()
     },
 
+    onPaste (event) {
+      this.valueText = (event.clipboardData || window.clipboardData).getData('text').trim()

```

Comment:
could both event.clipboardData and window.clipboardData ever be null? It might be safer to make sure that they aren't before calling getData otherwise we'll throw an exception here.

Reply:
This is a good suggestion, will add that check.